### PR TITLE
Components index.js manipulation

### LINKED ISSUE #8: Generating a new component manipulates index.js of components directory
If I have some blank lines in my `index.js` file of components directory, they disappear when I create a new component by CLI

Before:
```
export { default as Comp1 } from 'Src/components/Comp1'
export { default as Comp2 } from 'Src/components/Comp2'

export { default as Comp3 } from 'Src/components/Comp3'
export { default as Comp4 } from 'Src/components/Comp4'

```
After:
```
export { default as Comp1 } from 'Src/components/Comp1'
export { default as Comp2 } from 'Src/components/Comp2'
export { default as Comp3 } from 'Src/components/Comp3'
export { default as Comp4 } from 'Src/components/Comp4'
export { default as Comp5 } from 'Src/components/Comp5'

```

Generating new component should only add new component to the `index` file instead of remove empty lines.

### PR DESCRIPTION
**Pull request by Flixow**
Fixes #8

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
 program
   .version(pjson.version, '-v, --version')
+
   .command('gc <component>')
   .option('-s, --style', 'With stylesheet')
   .option('-f, --functional', 'Create functional component')
   .option('-c, --withConnect', 'Wrap with redux connect')
   .action(generateComponent);
 
+program
```
Comment (gh3r) [inline index.js]: why you use program here? it is already in line 7

File: package.json [hunk 2]
```diff
@@ -15,13 +15,16 @@
...
   "license": "MIT",
   "dependencies": {
+    "chai": "^4.1.2",
     "colors": "^1.3.2",
     "commander": "^2.18.0",
     "fs-extra": "^7.0.0",
-    "replace": "^1.0.0"
+    "mocha": "^5.2.0",
+    "replace": "^1.0.0",
+    "sinon": "^6.3.4"
```
Comment (gh3r) [inline package.json]: shouldn't we use exact version of packages?

File: index.js (hunk 1 above)
Comment (Flixow) [reply index.js]: For better readability as it's another command

File: package.json (hunk 2 above)
Comment (Flixow) [reply package.json]: Solved in #10

**Comment by Flixow** [comment]
Comments will be applied in #13

**Pull request closed without merging**